Examples/Swift/ViewController.swift

```diff
@@ -159,6 +159,7 @@ class ViewController: UIViewController, MGLMapViewDelegate {
         
         viewController.routeController.snapsUserLocationAnnotationToRoute = true
         viewController.voiceController?.volume = 0.5
+        viewController.navigationDelegate = self
```

Comment:
At the point where `NavigationViewController.navigationMapView(_:shapeDescribing:)` attempts to call `ViewController.navigationMapView(_:shapeDescribing:)`, is NavigationViewController’s `navigationController` property still set?